proto/cosmos/gov/v1beta1/genesis.proto

```diff
@@ -5,7 +5,7 @@ package cosmos.gov.v1beta1;
 import "gogoproto/gogo.proto";
 import "cosmos/gov/v1beta1/gov.proto";
 
-option go_package = "github.com/cosmos/cosmos-sdk/x/gov/types";
+option go_package = "github.com/cosmos/cosmos-sdk/x/gov/legacy/v040";
```

Comment:
We need gov@v1beta1 still for migrations. I changed their generated path to the legacy folder.

Reply:
ah, we need it to deserialize Anys?

Reply:
Yeah, old proposals content have e.g. `cosmos.gov.v1beta1.Textproposal`